Bug 1116803 - [rtl] [contacts] when adding new contact the add field and remove field buttons are not coresponding to select +autoland

anefzaoui opened the pull request.

try-server-hook commented:
[<img src="https://avatars.githubusercontent.com/u/1832078?v=3" alt="anefzaoui" width="50px" height="50px" /> Ahmed Nefzaoui (anefzaoui)](https://github.com/anefzaoui) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=b21502df2f3e)

Commit:
Bug 1116803 - [rtl] [contacts] main two contacts app UIs: Contacts List and Add new contact page lack RTL support

 Author:    Ahmed Nefzaoui <nefzaoui.ahmed@hotmail.com>
 Date:      Mon Jan 12 12:31:05 2015 -0800

try-server-hook commented:
[<img src="https://avatars.githubusercontent.com/u/1832078?v=3" alt="anefzaoui" width="50px" height="50px" /> Ahmed Nefzaoui (anefzaoui)](https://github.com/anefzaoui) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=864beb220590)

try-server-hook commented:
[<img src="https://avatars.githubusercontent.com/u/1832078?v=3" alt="anefzaoui" width="50px" height="50px" /> Ahmed Nefzaoui (anefzaoui)](https://github.com/anefzaoui) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=22218e9a0d90)

arcturus commented on apps/communications/contacts/style/app.css:
```diff
@@ -637,4 +603,49 @@ html[dir="rtl"] ::-moz-placeholder {
+
+html[dir="rtl"] .contact-checkbox {
+  left: unset;
+  right: 0 !important;
```
Can we get ride of this?

anefzaoui replied:
Unfortunately we can't :( Not adding !important will still make the original property a higher priority, thus it's as if we didn't fix it with this rule.

try-server-hook commented:
[<img src="https://avatars.githubusercontent.com/u/1832078?v=3" alt="anefzaoui" width="50px" height="50px" /> Ahmed Nefzaoui (anefzaoui)](https://github.com/anefzaoui) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=604d1c6ea233)

try-server-hook commented:
[<img src="https://avatars.githubusercontent.com/u/1832078?v=3" alt="anefzaoui" width="50px" height="50px" /> Ahmed Nefzaoui (anefzaoui)](https://github.com/anefzaoui) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=f5a7f9c8bfa9)

try-server-hook commented:
[<img src="https://avatars.githubusercontent.com/u/1832078?v=3" alt="anefzaoui" width="50px" height="50px" /> Ahmed Nefzaoui (anefzaoui)](https://github.com/anefzaoui) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=421e6ea49bd1)
